src/cosmos/Swap.fixture.tsx

```diff
@@ -47,8 +47,18 @@ function Fixture() {
   const [darkMode] = useValue('darkMode', { defaultValue: false })
   useEffect(() => setTheme((theme) => ({ ...theme, ...(darkMode ? darkTheme : lightTheme) })), [darkMode, setTheme])
 
-  const jsonRpcEndpoint = useJsonRpcEndpoint()
-  const connector = useProvider()
+  const jsonRpcEndpoint = INFURA_NETWORK_URLS
+
+  const defaultNetwork: string | undefined = useOption('defaultChainId', {
+    options: Object.values(SupportedChainId).filter((id) => typeof id !== 'number') as string[],
+    defaultValue: 'MAINNET',
+  })
+  function getValueByKeyForNumberEnum(value: string): number {
+    return Object.entries(SupportedChainId).find(([key, val]) => key === value)?.[1] as number
+  }
+  const defaultChainId = defaultNetwork ? getValueByKeyForNumberEnum(defaultNetwork) : undefined
```

Comment:
honestly all this is just so that the Cosmos viewer shows the network name instead of the chainId -- a little easier to play around with

<img width="230" alt="Screen Shot 2022-07-21 at 1 53 22 PM" src="https://user-images.githubusercontent.com/27475332/180281229-6e45c9ac-3472-4844-911c-19604306b035.png">

Reply:
but can be removed -- we'll just have the Cosmos viewer have a chainId dropdown instead